pandunia-loge.csv

```diff
@@ -1597,7 +1597,7 @@ Himalaya||||Himalayas||Himalaya|||||||||||||Himalajo|Himalaja|Himalaje
 hin|||hin:-हीन (hīn), ben:-হীন (-hin), guj:-હીન (-hīn)|lack (miss, be without)||faltar (sin)|sem|без|||||||||||sen (manki)|puuttua (ilman, vailla)|bez
 hin di||||lacking (-less)|||||||||||||||ne havanta (sen~a)|ilman oleva (-ton)|
 hina|bio|Lawsonia inermis|ara: حِنَّاء‎ (ḥinnāʾ), eng:henna, fra:henné, spa:alheña, por:hena, rus:хна (hna), hin:हिना (hinā)|henna (alcanna)|henné|alheña (henna)|hena|хна|حِنَّاء‎|||||मेंहदी (हिना)|মেহেদী|||kına||hennapensas|
-hindi yum|yum 049|In|eng:indium, fra:indium, spa:indium, por:índio, rus:индий, zho:铟 (yīn), jpn:インジウム, kor:인듐, vie:indi, hin:इण्डियम, ben:ইন্ডিয়াম, may:indium, swa:indi, ara: انديوم|indium|indium|indium|índio|индий|انديوم|铟|インジウム|인듐|indi|इण्डियम|ইন্ডিয়াম|indium|indi|i̇ndiyum|indio|indium|ind
+indi yum|yum 049|In|eng:indium, fra:indium, spa:indium, por:índio, rus:индий, zho:铟 (yīn), jpn:インジウム, kor:인듐, vie:indi, hin:इण्डियम, ben:ইন্ডিয়াম, may:indium, swa:indi, ara: انديوم|indium|indium|indium|índio|индий|انديوم|铟|インジウム|인듐|indi|इण्डियम|ইন্ডিয়াম|indium|indi|i̇ndiyum|indio|indium|ind
```

Comment:
I propose we use hindi and change the following lines to hindi din ja, hindi basha and hindi desha.

Reply:
that makes sense to me.